src/hera/shared/_global_config.py

```diff
@@ -7,7 +7,7 @@
 from typing import Any, Callable, Dict, List, Optional, Type, TypeVar, Union
 
 from hera.auth import TokenGenerator
-from hera.shared._pydantic import BaseModel, root_validator
+from hera.shared._pydantic import BaseModel, get_fields, root_validator
```

Comment:
this `get_fields` thing I did mainly to avoid littering all the existing raw usages of `__fields__` with `# type: ignore` because there's version specific linter error I was seeming to get.